core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -497,13 +498,28 @@ class MultiCloudWorkspaceService(override val ctx: RawlsRequestContext,
       throw new RawlsExceptionWithErrorReport(ErrorReport(StatusCodes.NotImplemented, "MC workspaces are not enabled"))
     }
 
+    assertBillingProfileCreationDate(profile)
+
     traceWithParent("createMultiCloudWorkspace", parentContext)(s1 =>
       createWorkspace(workspaceRequest, profile, s1) andThen { case Success(_) =>
         createdMultiCloudWorkspaceCounter.inc()
       }
     )
   }
 
+  def assertBillingProfileCreationDate(profile: ProfileModel): Unit = {
+    val previewDate = new DateTime(2023, 9, 12, 0, 0, DateTimeZone.UTC)
+    val isTestProfile = profile.getCreatedDate() == null || profile.getCreatedDate() == ""
```

Comment:
This is to get around billing profiles in our unit tests? It'd be nice to not have to include this if it's not too painful to add created dates, but I don't have an actual sense for how many creation dates would actually have to be added

Reply:
Yes, it is to get around unittest where we create a BillingProfile but don't give it a created date.

![image](https://github.com/broadinstitute/rawls/assets/484484/6538aed3-fd07-4e6a-bc8c-479c628ce3fe)